File: 프로그래머스/1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    answer = ''
    
    for ch in s:
        # 공백
        if ch == ' ':
            answer += ' '
        
        # 소문자
        elif ch.islower():
            answer += chr((ord(ch) - ord('a') + n) % 26 + ord('a'))
        
        # 대문자
        else:
            answer += chr((ord(ch) - ord('A') + n) % 26 + ord('A'))
    
    return answer

File: 프로그래머스/1/Java/12926. 시저 암호/시저 암호.java
class Solution {
    public String solution(String s, int n) {
        String answer = "";

        for (char c : s.toCharArray()) {
            if (c == ' ') {
                answer += ' ';
            } 
            else if (c >= 'A' && c <= 'Z') {
                answer += (char) ((c - 'A' + n) % 26 + 'A');
            } 
            else {
                answer += (char) ((c - 'a' + n) % 26 + 'a');
            }
        }

        return answer;
    }
}